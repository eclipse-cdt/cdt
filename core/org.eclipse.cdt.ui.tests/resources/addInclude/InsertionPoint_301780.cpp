#ifndef TST_H_
#define TST_H_

XXX var;

#endif /* TST_H_ */
