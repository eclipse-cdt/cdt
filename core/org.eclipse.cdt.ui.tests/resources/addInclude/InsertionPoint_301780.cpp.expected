#ifndef TST_H_
#define TST_H_
#include "InsertionPoint_301780.h"

XXX var;

#endif /* TST_H_ */
